Jint/Runtime/Debugger/CallFrame.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using Esprima;
+using Esprima.Ast;
+using Jint.Native;
+using Jint.Runtime.CallStack;
+using Jint.Runtime.Environments;
+
+namespace Jint.Runtime.Debugger
+{
+    public class CallFrame
```

Comment:
should be sealed by default and opened when someone has good reasoning, less to worry about how being used in the wild